rskj-core/src/test/java/co/rsk/core/BlockHeaderTest.java

```diff
@@ -492,17 +528,46 @@ public void fullEncodedV1IsTheSameAsEncodedForHeaderMessageButLogsBloomAndEdges
         headerV1.setLogsBloom(logsBloom);
 
         RLPList fullEncoded = RLP.decodeList(headerV1.getFullEncoded());
-        RLPList encodedForHeaderMessage = RLP.decodeList(headerV1.getEncodedForHeaderMessage());
+        RLPList encodedCompressed = RLP.decodeList(headerV1.getEncodedCompressed());
+
+        // extension data takes the element of logs bloom
+        // version element is not existent, is inside the rlp list od the extension data
+        // edges is not existent
+        int sizeDifference = 2;
+        Assertions.assertEquals(fullEncoded.size() - sizeDifference, encodedCompressed.size());
+
+        for (int i = 0; i < encodedCompressed.size(); i++) {
+            int j = i < 16 ? i : i + sizeDifference; // keep comparing elements jumping version and edges
+            if (i != 6) {
+                // logs bloom field
+                Assertions.assertArrayEquals(fullEncoded.get(j).getRLPData(), encodedCompressed.get(i).getRLPData());
+            }
+        }
+    }
 
-        Assertions.assertEquals(fullEncoded.size() - 1, encodedForHeaderMessage.size());
+    @Test
+    public void compressedEncodingV1HasSameRLPSizeAsFullEncodedV0WithoutEdges () {
+        // this test is added to assert that the rlp header size does not change
+        // in the hard fork, assuming both RSKIP 351 and RSKIP 144 are activated
+        // together
+        byte[] logsBloom = new byte[Bloom.BLOOM_BYTES];
```

Comment:
vars like this one or `edges` that are repeated initialization, could be declared static with some meaninful name